doc/building-and-deploying-the-website.md

```diff
@@ -0,0 +1,22 @@
+## Building and Deploying (Automated)
+
+### Testing
+
+After you are finished developing and making your changes, make sure to test them. Run:
+
+	npm test
+
+`npm test` runs both [eslint] and [mocha] tests. If your tests pass, commit and push your work to Github.
+
+### Travis 
+
+[Travis] automatically builds and publishes the website on every change. Travis will build the full website for you by running `gulp build --prod` under the hood. Travis also uses [SVN] to update, copy, add, and commit the new changes over to the website. Committing to svn can only occur once the commit has been merged to master. 
+
+Lastly, Travis also runs `npm test` and will notify you if any of your `eslint` or `mocha` tests are failing. When Travis is done building and deploying, send a pull request and ask for a review.
```

Comment:
Why the seperate "Testing" paragraph if Travis automatically runs the tests? Is this a safeguard somehow?